doc/manual/rule_uid_schema.md

```diff
@@ -0,0 +1,166 @@
+ (?P<STANDARD>([a-z]+))?:
+ (?P<STD_VERSION>([0-9]+(\.[0-9]+)+))?:
+ (?P<RULESET>(([a-z][\w_]*)\.)*)(?P<NAME>[a-z][\w_]*)
+$
+```
+
+### ASAM Concepts
+
+ * Emanating Entity: `asam.net`
+ * Standards:
+   * Quality Checker Framework: `asam.net:qc`
+   * OpenDRIVE: `asam.net:xodr`
+     * v1.4.0: `asam.net:xodr:1.4.0`
+     * v1.5.0: `asam.net:xodr:1.5.0` 
+     * v1.6.0: `asam.net:xodr:1.6.0`
+     * v1.6.1: `asam.net:xodr:1.6.1`
+     * v1.7.0: `asam.net:xodr:1.7.0`
+     * v1.8.0: `asam.net:xodr:1.8.0`
+   * OpenSCENARIO XML: `asam.net:xosc`
+     * v1.0.0: `asam.net:xosc:1.0.0` 
+     * v1.1.0: `asam.net:xosc:1.1.0`
+     * v1.1.1: `asam.net:xosc:1.1.1`
+     * v1.2.0: `asam.net:xosc:1.2.0` 
+   * OpenSCENARIO DSL: `asam.net:osc`
+     * v2.0.0: `asam.net:osc:2.0.0`
+
+Rule full names are defined in domain-specific Quality Checker subgroups. Third
+party emanating entities should use the standard and definition setting
+concepts defined by ASAM. 
+
+## Example
+
+Lets start with an example for a rule UID, and lets break down its concepts. The
+example references a rule listed in the subsection **Rules** of one of the ASAM
+Standards for OpenDRIVE. The Rules subsections were introduced for the first
+time in revision 1.6 of the standard. **Please notice that the full name of the
+rule may change in the future**.
+
+> [ASAM OpenDRIVE Standard 1.6.0, Chapter 7.2 - Road Reference
+> Line](https://releases.asam.net/OpenDRIVE/1.6.0/ASAM_OpenDRIVE_BS_V1-6-0.html#_road_reference_line),
+> subsection Rule, first point: 
+> > **Each road shall have a reference line**
+
+we can identify the following concepts:
+
+ * **emanating entity**: ASAM `asam.net`
+ * **standard**: OpenDRIVE `xodr`
+ * **definition settings**: the rule firstly appeared in `1.6.0`, the first
+   revision to introduce the subsections rules
+ * **Full name**:
+    * **rule-set**: Road Planar Geometry definitions `road.planview.geometry`
+    * **name**: summarizes the content of the rule, e.g. `ref_line_exists`. It
+      is always the last group in dot-notation strings.
+
+The complete UID would be something like:
+
+```
+asam.net:xodr:1.6.0:road.planview.geometry.ref_line_exists
+-------- ---- ----- ---------------------- ---------------
+ |        |    |     |                      |             
+ entitity | version  |                    name
+       standard    rule-set
+```
+
+Regarding the validation for the schema, **only the the emanating concept and
+the name are required**. Third party entities are not required to declare
+standard, definition settings, and rule-set. Still, separation
+characters cannot be omitted. ASAM rules shall always define all concepts.
+
+> Example: if `example.com` entity defines it's own rules, it may be something
+> like: `example.com:::custom_rule` or `example.com:qc::custom_rule` and the
+> UID should be considered valid. **However, entity should use ASAM standard and
+> definition setting concepts in its custom rules.**
+
```

Comment:
Don't understand the last part `**However, entity should use ASAM standard and
> definition setting concepts in its custom rules.**`

Did you want to say again that ASAM shall always use all concepts? If yes, then in might be simply removed. And the sentence `ASAM rules shall always define all concepts.` from the chapter above can be placed below the example for third-party rules.

Reply:
I want to suggest that third party entities (companies or organizations) should use all the other concept if possible. For example even if `example.com` can create the uid `example.com:::custom_rule` for a custom rule on OpenDRIVE 1.7, the rule uid should be preferably `example.com:xodr:1.7.0:custom_rule`